components/edit-collective/VirtualCard.js

```diff
@@ -177,12 +177,20 @@ ActionsButton.propTypes = {
   editHandler: PropTypes.func,
 };
 
-const getLimitString = ({ spend_limit, spend_limit_duration }) => {
-  const value = formatCurrency(spend_limit, 'USD');
-  if (spend_limit === 0) {
+const getLimitString = ({ spendingLimitAmount, spendingLimitInterval }) => {
+  const value = formatCurrency(spendingLimitAmount, 'USD');
+  if (!spendingLimitAmount) {
     return <FormattedMessage id="VirtualCards.NoLimit" defaultMessage="No Limit" />;
   }
-  switch (spend_limit_duration) {
+  switch (spendingLimitInterval) {
+    case 'DAILY':
+      return (
+        <Fragment>
+          <FormattedMessage id="VirtualCards.LimitedTo" defaultMessage="Limited to" />
```

Comment:
The ids are not required for new strings as per, https://github.com/opencollective/opencollective/issues/4697#issuecomment-921858385

```suggestion
          <FormattedMessage defaultMessage="Limited to" />
```